isofit/utils/extractions.py

```diff
@@ -152,9 +152,11 @@ def extractions(
 
     nl, nb, ns = [int(meta[n]) for n in ("lines", "bands", "samples")]
     img_mm = in_img.open_memmap(interleave="bip", writable=False)
+    del in_img, img_mm
 
```

Comment:
We can remove the img_mm opening here entirely, and then not delete it as well.

Reply:
Indeed, it is not used at all. -> https://github.com/isofit/isofit/pull/695/commits/efab51bded055c86941abd8c4eada93ab9fdc1c8